packages/-ember-data/tests/integration/record-arrays/adapter-populated-record-array-test.js

```diff
@@ -107,9 +107,8 @@ module('integration/record-arrays/adapter_populated_record_array - AdapterPopula
       },
     };
 
-    run(() => {
-      recordArray._setInternalModels(store._push(payload), payload);
-    });
+    let results = store.push(payload);
+    recordArray._setIdentifiers(results.map(r => r._internalModel.identifier), payload);
```

Comment:
It might be worth refactoring `getRecord()` to return out an identifier as a top level property.

Reply:
a helper function for tests I think exists for getting the identifier from the record